CONTRIBUTING.md

```diff
@@ -41,9 +41,14 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests!
 
-1. Fork the repo and create your branch from `master`.
+If you need help with GIT or our workflow, please ask on [Gitter.im](https://gitter.im/netlify/NetlifyCMS). We want your contributions even if you're just learning GIT. Our maintainers are happy to help!
```

Comment:
This looks awesome!

Reply:
I was thinking leading with it was better. If people are experienced and in a hurry, they're likely to just notice the heading and skip down to the list. We tend to bypass blocks of text when speed reading.

Reply:
If they trying to make sure they're doing right and paying attention I figured it would be good to put the encouragement up front.

Reply:
@dopry I like that reasoning, and I agree.